.github/workflows/reusable_compatibility.yml

```diff
@@ -110,7 +110,7 @@ jobs:
       run: >
         UMF_LOG="level:warning;flush:debug;output:stderr;pid:no" 
         LD_LIBRARY_PATH=${{github.workspace}}/latest_version/build/lib/ 
-        ctest --output-on-failure
+        ctest --output-on-failure -E "umf-mempolicy" # disable tests that rely on internal structures
```

Comment:
@bratpiorka Is such a comment enough?

Reply:
This test directly dereferences internal pointers and uses specific fields without using the UMF API. If the structures are changed (as is the case in this PR) this will no longer work,

Reply:
@ldorau Is this explanation sufficient, or would you like to modify this comment? If you prefer the latter, please suggest changes.